input/ig.xml

```diff
@@ -398,6 +398,14 @@
 			<description value="Active prescription for Reaptan 10 mg/10 mg (perindopril arginine/amlodipine) tablet, 10."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-medicationrequest"/>
 		</resource>
+		<resource>
+			<reference>
+				<reference value="MedicationRequest/simvastatin"/>
+			</reference>
+			<name value="MedicationRequest - active prescription, Simvastatin"/>
+			<description value="Active prescription for Simvastatin 40 MG Disintegrating Oral Tablet."/>
+			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-medicationrequest"/>
+		</resource>
```

Comment:
Missing additions of the new examples e.g.:
Observation/blood-group
Observation/blood-group-panel
Observation/rh-status

Reply:
Added